example-adapter/Setting-up-Development-Environment.md

```diff
@@ -2,24 +2,24 @@ This document describes how a developer's workstation can be setup.
 
 ### Software Requirements
 
-* Linux or Windows
-* Java 7
-* Tomcat 6 or 7 or 8
+* Linux / Windows / MacOS
+* Java 8 or Java 11
 * Maven 3.x
+* Docker (*optional*)
 
 ### Getting the code
 
-The example adapter project is a part of the XRd4J git repository. You can access the source code by cloning it:
+The Example Adapter project is a part of the XRd4J git repository. You can access the source code by cloning it:
 
 ```
```

Comment:
This could have the language type set as well. Probably "bash" would be good.

Reply:
Language type bash added.